docs/expand-pvc.md

```diff
@@ -10,13 +10,13 @@
 
 ## Prerequisite
 
-- For filesystem resize to be supported for your kubernetes cluster, the
+- For filesystem expansion to be supported for your kubernetes cluster, the
   kubernetes version running in your cluster should be >= v1.15 and for block
   volume resize support the kubernetes version should be >=1.16. Also,
   `ExpandCSIVolumes` feature gate has to be enabled for the volume resize
```

Comment:
changes resize to expand in other places also

Reply:
@Madhu-1 sharp.. thanks.. updated in the other places too.